packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/index.tsx

```diff
@@ -67,8 +65,8 @@ export function WalletStatusDialog(props: WalletStatusDialogProps) {
         <InjectedDialog title="Mask Network" open={open} onClose={closeDialog} maxWidth="sm">
             <DialogContent className={classes.content}>
                 <Typography className={classes.subTitle}>{t('wallets')}</Typography>
-                <WalletStatusBox isDashboard={props.isDashboard} />
-                {!props.isDashboard && <ApplicationBoard />}
+                <WalletStatusBox />
+                {!isDashboardPage() && <ApplicationBoard />}
```

Comment:
🤔 Should we move this file out of SNSAdaptor folder since it is shared with dashboard?